src/ceph-disk/ceph_disk/main.py

```diff
@@ -2756,6 +2756,31 @@ def set_data_partition(self):
                       self.args.data)
             self.partition = self.create_data_partition()
 
+    def set_journal_owner(self, journal):
```

Comment:
@tengjie001 (NIT) Please consider to Keep PR title < 50 characters maybe for future PRs
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits